test/transcoding/SPV_INTEL_kernel_attributes/intel_fpga_function_attributes.ll

```diff
@@ -53,6 +53,7 @@
 ; CHECK-SPIRV-DAG: Decorate [[FUNCENTRY]] InitiationIntervalINTEL 10
 ; CHECK-SPIRV-DAG: Decorate [[FUNCENTRY]] MaxConcurrencyINTEL 12
 ; CHECK-SPIRV-DAG: Decorate [[FUNCENTRY]] PipelineEnableINTEL 0
+; CHECK-SPIRV: Decorate [[FUNCENTRY2:[0-9]+]] PipelineEnableINTEL 1
```

Comment:
FUNCENTRY2 is not used later, should it be added to check, that decoration is applied to the correct entry point?

Reply:
Added it in [a75b8f7](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1323/commits/a75b8f7e4fd53f9d10dd9e19810dd3b5bb0e1ff8)